### ISSUE TITLE
Fix CssSyntaxError for unclosed comment

### ISSUE THREAD

**Issue by XhmikosR** [stylelint/stylelint#3955]
> What CSS is needed to reproduce the bug?

```less
// stylelint-disable value-keyword-case

//** Background color for `<body>`.
```

> What stylelint configuration is needed to reproduce the bug?

https://github.com/twbs/bootstrap/blob/v3-dev/grunt/.stylelintrc

> Which version of stylelint are you using?

`9.10.1`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

Via grunt-stylelint

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

It's related to Less

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

```shell
$ grunt stylelint
Running "stylelint:dist" (stylelint) task

less/variables.less
 3:2  ×  Unclosed comment   CssSyntaxError

✖ 1 problem
```

Note that this doesn't happen with v9.6.0.

Maybe it was intended to break I just wanted to report it. Basically the `//**` confuses the parser thinking it's an unclosed `/* comment */`.

**Comment by jeddy3** [comment]
@XhmikosR Thanks for the report and for using the template. I can reproduce this. 

I've raised an issue upstream in the Less parser stylelint uses: https://github.com/shellscape/postcss-less/issues/135

If you've any experience with parsers, feel free to help fix this issue upstream.

**Comment by XhmikosR** [comment]
Thanks! I wasn't sure if it's a parser issue that's why I reported it here. I'm gonna watch the upstream discussion.

This might be a regression in postcss-less 3.x, but I'm not 100% sure. I know for sure stylelint 9.6.0 works fine.

**Comment by jwilsson** [comment]
This is fixed upstream in `postcss-less@3.1.4` so a fresh `npm install` should take care of it automatically.
